src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
-  margin: 0;
-}
+html {
+  font-family: Roboto, sans-serif;
+  }
+
+  body {
+    margin: 0;
+    }
+
+    .header {
+      align-items: center;
+      justify-content: space-between;
+      margin: 0;
+      padding: 0 50px;
+      display: flex;
+      }
```

Comment:
Fix all cases

```suggestion
      display: flex;
    }
```

Reply:
Please fix code styling at css file as mentors asked
It should look like
```
.selector1 {
  rule1: value1;
  rule2: value2;
}

.selector2 {
  rule1: value1;
  rule2: value2;
  
  &__selector {
    rule1: value1;
    rule2: value2;
  }
}

.selector3 {
  rule3: value1;
  rule3: value2;
}
```